src/actions/actions.ts

```diff
@@ -590,13 +590,15 @@ class CommandInsertInInsertMode extends BaseCommand {
     if (char === "<backspace>") {
       if (position.character === 0) {
         if (position.line > 0) {
+          const prevEndOfLine = position.getPreviousLineBegin().getLineEnd();
```

Comment:
:100: This fix is truly representative. We should be carefull with `TextEditor.delete`, etc as they have kind of `side effect`.